### ISSUE TITLE
plotly.js pseudo-html docs

### ISSUE THREAD

**Issue by alexcjohnson** [plotly/documentation#766]
The html-like system we have for formatting text in plotly.js is getting more and more complicated, and is used in more and more places. see https://github.com/plotly/plotly.js/pull/1726 for the latest.

We should have a dedicated page to collect this, and somehow link it to the reference for each of the different attributes that can use it:

> Our pseudo-html applies to really anywhere we display text: annotations, plot and axis titles, trace names. Also tick labels (categories, ticktext, tickprefix, ticksuffix...) and trace text and hovertext

cc @cldougl @etpinard

**Comment by cldougl** [comment]
good idea, we can have a text formatting page and link that on the general scatter text / hovertext / axes formatting pages as well.
